db/db.pl

```diff
@@ -5489,6 +5489,80 @@ sub lookupsUpdate
 }
 ################################################################################
 
+################################################################################
+sub notifiersCreate
+{
+  my $settings = '
+{
+  "settings": {
+    "index.priority": 30,
+    "number_of_shards": 1,
+    "number_of_replicas": 0,
+    "auto_expand_replicas": "0-3"
+  }
+}';
+
+  logmsg "Creating notifiers_v40 index\n" if ($verbose > 0);
+  esPut("/${PREFIX}notifiers_v40?master_timeout=${ESTIMEOUT}s", $settings);
+  esAlias("add", "notifiers_v40", "notifiers");
+  notifiersUpdate();
+}
+
+sub notifiersUpdate
+{
+    my $mapping = '
+{
+  "_source": {"enabled": "true"},
+  "dynamic": "true",
+  "properties": {
```

Comment:
unless there is a reason not to, I think we should also add a dynamic string template, it basically makes anything not listed automatically a keyword instead of ending up with text fields. See fields_v30 and others. Still need to definitely list out the date fields.

    my $mapping = '
{
  "_source": {"enabled": "true"},
  "dynamic_templates": [
    {
      "string_template": {
        "match_mapping_type": "string",
        "mapping": {
          "type": "keyword"
        }
      }
    }
...

Reply:
you can leave them there, it is slight more efficient for ES if the fields you know about are listed.